### ISSUE TITLE
UI: (13.C) Crossfeed Site Insufficient Contrast on Search (CRASM-370)

### ISSUE THREAD

**Issue by hawkishpolicy** [cisagov/XFD#397]
# 🐛 Summary #

Insufficient contrast is provided for “Search a domain, vuln, port, service, IP”. The color difference of text and its background should have a contrast ratio of at least 4.5:1. Using contrast analyzers, spot test various text and backgrounds throughout the site or application to ensure a contrast ratio of at least 4.5:1. When insufficient contrast issues are found, adjust either one or the other color to meet a minimum of 4.5:1 ratio.

## Any helpful log output or screenshots ##

![image](https://github.com/cisagov/XFD/assets/123108455/61231404-1a3f-472d-bcaf-d1f39fa48117)
